src/vs/platform/environment/node/argv.ts

```diff
@@ -131,12 +131,17 @@ export const OPTIONS: OptionDescriptions<Required<NativeParsedArgs>> = {
 	'locate-shell-integration-path': { type: 'string', args: ['bash', 'pwsh', 'zsh'] },
 
 	// {{SQL CARBON EDIT}} Start
-	'command': { type: 'string', alias: 'c', cat: 'o', args: 'command-name', description: localize('commandParameter', 'Name of command to run') },
-	'database': { type: 'string', alias: 'D', cat: 'o', args: 'database', description: localize('databaseParameter', 'Database name') },
-	'server': { type: 'string', alias: 'S', cat: 'o', args: 'server', description: localize('serverParameter', 'Server name') },
-	'user': { type: 'string', alias: 'U', cat: 'o', args: 'user-name', description: localize('userParameter', 'User name for server connection') },
-	'aad': { type: 'boolean', cat: 'o', description: localize('aadParameter', 'Use Azure Active Directory authentication for server connection') },
-	'integrated': { type: 'boolean', alias: 'E', cat: 'o', description: localize('integratedAuthParameter', 'Use Integrated authentication for server connection') },
+	'aad': { type: 'boolean', cat: 'o', description: localize('aadParameter', 'Use Azure Active Directory authentication, this option is depcrecated - use \'authenticationType\' instead.') },
+	'applicationName': { type: 'string', alias: 'A', cat: 'o', description: localize('applicationNameParameter', 'Supports providing applicationName that will be used for connection profile app name.') },
+	'authenticationType': { type: 'string', alias: 'L', cat: 'o', description: localize('authenticationTypeParameter', 'Provide authentication mode to be used. Accepted values: AzureMFA, SqlLogin, Integrated, etc.') },
```

Comment:
Some of the aliases seem odd, like why is "L" authenticationType? Generally the aliases should at least be a letter within the option name (and ideally the first letter of one of the sub-words, e.g. for this I'd suggest `T` since `A` is already taken.)